app/main.py

```diff
@@ -1,8 +1,54 @@
 class Car:
-    # write your code here
-    pass
-
+    def __init__(self, comfort_class, clean_mark, brand):
```

Comment:
Missing type annotation for function argument 'comfort_class', 'clean_mark', 'brand'. Missing return type annotation for special method